compute/ibm_vpc.py

```diff
@@ -0,0 +1,451 @@
+import logging
+import re
+import socket
+from datetime import datetime, timedelta
+from time import sleep
+from typing import List
+
+from ibm_cloud_networking_services import DnsSvcsV1
+from ibm_cloud_sdk_core.api_exception import ApiException
+from ibm_cloud_sdk_core.authenticators import IAMAuthenticator
+from ibm_vpc import VpcV1
+
+from ceph.parallel import parallel
+
+LOG = logging.getLogger()
+
+socket.setdefaulttimeout(280)
+
+
+def get_ibm_service(accessKey: str):
+    """
+    Get ibm service
+    Args:
+        accessKey    The access key(API key) of the user.
+    """
+    authenticator = IAMAuthenticator(accessKey)
+    service = VpcV1(authenticator=authenticator)
+    service.set_service_url("https://au-syd.iaas.cloud.ibm.com/v1")
```

Comment:
I believe it would be good if we got this endpoint from the global configuration file.

Reply:
Done